aiida/repository/repository.py

```diff
@@ -486,10 +486,10 @@ def copy_tree(self, target: Union[str, pathlib.Path], path: FilePath = None) ->
             target = pathlib.Path(target)
 
         if not isinstance(target, pathlib.Path):
-            raise TypeError('path is not of type `str` nor `pathlib.Path`.')
+            raise TypeError(f'path `{path}` is not of type `str` nor `pathlib.Path`.')
 
         if not target.is_absolute():
-            raise TypeError('provided target is not an absolute path.')
+            raise TypeError(f'provided target `{target}` is not an absolute path.')
```

Comment:
Why are you fixing this here instead of modifying the PR #5114 ? I mean, it is still up, wouldn't it be cleaner to have that there?

Reply:
Moved to 5114